### ISSUE TITLE
Is rsync just missing from the wiki or was it removed?

### ISSUE THREAD

**Issue by alejoluc** [bmatzelle/gow#175]
I was checking out this software and went to the wiki where it lists the executables, to see if it includes rsync, an indispensable tool:

https://github.com/bmatzelle/gow/wiki/executables_list

It says it was last edited in November 2014. So I googled "gow rsync" and it actually took me to an issue in this very same repo:

https://github.com/bmatzelle/gow/issues/15

Where one of the answers says:

"
Came here to say that it is part of Gow 0.5 but it isn't on the wiki list but you beat me too it. However, the version is quite old. 2.6.3 vs upstream 3.0.9.
"

and then the owner of the repo also comments, saying that he has added it to the wik. However, now rsync is nowhere to be found. An user noted this in april 2014, but he received no answer.

rsync is amazing, and it would be a big plus for gow to have it.

**Comment by kk17** [comment]
I found rsync missing too.

**Comment by bmatzelle** [comment]
I haven't been able to find a good binary with source code.

**Comment by divinity76** [comment]
agreed. i didn't switch from cygwin to gow, and this is part of the reason (its missing `netcat` and `rsync` and `ssh` - ps, cygwin has all 3)
